appveyor.yml

```diff
@@ -6,15 +6,22 @@ branches:
 
 cache:
   # Note that we don't use cache for local Maven repository - see https://github.com/SonarSource/sonar-java/pull/525
-  - C:\ProgramData\chocolatey\bin -> appveyor.yml
-  - C:\ProgramData\chocolatey\lib -> appveyor.yml
-  - C:\bin\apache-maven-3.2.5 -> appveyor.yml
+  - C:\Users\appveyor\maven -> appveyor.yml
 
 install:
   - git submodule update --init --recursive
   - set MAVEN_VERSION=3.2.5
-  - choco install maven -version %MAVEN_VERSION%
-  - set PATH=%PATH%;C:\bin\apache-maven-%MAVEN_VERSION%\bin
+  # Note that indentation is important here:
+  - ps: |
+      Add-Type -AssemblyName System.IO.Compression.FileSystem
```

Comment:
@dbolkensteyn You're right, I was thinking about this: I suppose that recent Windows machines do have PowerShell by default, while according to http://www.appveyor.com/docs/installed-software#tools `unzip` comes from 7-zip and `curl` comes as a transitive dependency of Git. So this variant can be copy-pasted easily into another environment, while another not. From here it is probably up to Java team to decide which way they'd like to maintain ( cc @benzonico ).